src/v3/src/components/AuthContainer/AuthContainer.tsx

```diff
@@ -18,9 +18,9 @@ import { FunctionComponent, h } from 'preact';
 import { useWidgetContext } from '../../contexts';
 import style from './style.module.css';
 
-const AuthContainer: FunctionComponent = ({ children }) => {
+const AuthContainer: FunctionComponent<{ hide?: boolean }> = ({ children, hide }) => {
```

Comment:
nit: since `hide` is initialized with a default value and always passed to the component, you can make this non-optional
```suggestion
const AuthContainer: FunctionComponent<{ hide: boolean }> = ({ children, hide }) => {
```